### ISSUE TITLE
cvmfs-contrib unavailable for ubuntu

### ISSUE THREAD

**Issue by astroclark** [cvmfs-contrib/cvmfs-contrib-release#7]
The cvmfs-contrib repository appears to be unavailable for Ubuntu, although I am able to install `cvmfs-x509-helper` in Ubuntu using the Debian 9 cvmfs-contrib repository by replacing the symlink to the ubuntu repo with one to the debian repo:

Try to install cvmfs-x509-helper:
```
➜  ~ apt install cvmfs-x509-helper
<snip>
The following packages have unmet dependencies: cvmfs-x509-helper : Depends: libvomsapi1 but it is not installable
```

Now follow instructions at https://cvmfs-contrib.github.io/ to enable cvmfs-contrib repositories.  Updating apt gives:
```
N: Ignoring 'cvmfs-contrib.list' in directory '/etc/apt/sources.list.d/' as it is not a regular file
```
Because the symlink:
```
/etc/apt/sources.list.d/cvmfs-contrib.list -> /usr/share/cvmfs-contrib-release/cvmfs-contrib-Ubuntu_18.04.list
```
points to a non-existent file
```
ls /usr/share/cvmfs-contrib-release/
cvmfs-contrib-Debian_8.0.list  cvmfs-contrib-Ubuntu_14.04.list  cvmfs-contrib-Ubuntu_17.04.list
cvmfs-contrib-Debian_9.0.list  cvmfs-contrib-Ubuntu_16.04.list
```
Further, the URLs for the other Ubuntu releases give a 404 error.  

Replacing the symlink to the non-existent Ubuntu repository with a link to the Debian 9 repository allows me to install, and use, cvmfs-x509-helper just fine.

**Comment by DrDaveD** [comment]
Fixed in #8 and built in the cvmfs-contrib repo.  Please test.

**Comment by astroclark** [comment]
Confirmed: tested on Ubuntu 18.04 by creating file and symlink:
```
➜  ~ ls -l /etc/apt/sources.list.d/cvmfs-contrib.list 
lrwxrwxrwx 1 root root 64 Sep 28 10:14 /etc/apt/sources.list.d/cvmfs-contrib.list -> /usr/share/cvmfs-contrib-release/cvmfs-contrib-Ubuntu_18.04.list
➜  ~ /bin/cat cvmfs-contrib.list                     
# Do not edit.  Remove symlink and make a copy in /etc/apt/sources.list.d
deb http://downloadcontent.opensuse.org/repositories/home:/cvmfs:/contrib/xUbuntu_18.04 ./
# deb http://downloadcontent.opensuse.org/repositories/home:/cvmfs:/contrib-testing/xUbuntu_18.04 ./
```
And successfully installed `cvmfs-x509-helper`:
```
➜  ~ apt-cache policy cvmfs-x509-helper              
cvmfs-x509-helper:
  Installed: 1.1.1-1
  Candidate: 1.1.1-1
  Version table:
 *** 1.1.1-1 500
        500 http://downloadcontent.opensuse.org/repositories/home:/cvmfs:/contrib/xUbuntu_18.04 ./ Packages
        100 /var/lib/dpkg/status
     1.0.1 500
        500 http://software.ligo.org/lscsoft/debian jessie/contrib amd64 Packages
```
